### ISSUE TITLE
Driver drives off too soon when filled from a silo

### ISSUE THREAD

**Issue by Bobster82** [Stephan-S/FS19_AutoDrive#822]
Hello,

I just found a new issue (maybe it has something to do with the 0,01 drive off commit)

When I have a driver loading goods from a silo, it doesnt fully load... I loads about 30% and then just drives off

edit: Its actually 25% and does it also on other buildings like a fermenter (global company)

edit2: I did some more testing and found something strange;
* When I use a truck (tatra phoenix 6x6) with krampe sbII 30/1070 it just loads normal to 100%
* When I use a tractor (fend 1050) with dolly and the krampe trailer, it drives off too soon
* When using the tractor with krampe bandit 750, it drives off too soon
* When using the truck with krampe bandit 750, it drives off too soon

edit3:
When I use a trailer to wait at a field (pickup and deliver mode), it waits for 1 or 2 seconds, then it goes to the unload destination. Usually it waits till the fill level (set in settings) is reached before taking off

**Comment by geo-luchs** [comment]
I've same issue after downloading today the github zipfile.
With download from 6.11.2019 it works

**Comment by Stephan-S** [comment]
Should be fixed again. That was my fault. Forgot about some sideeffect there

**Comment by Bobster82** [comment]
> Should be fixed again. That was my fault. Forgot about some sideeffect there

Thanks!
Its almost fixed as it was... The trailers are filling up again and waiting till full... als waiting on the field...
But the problem is that they wait till they are 100% full... before it was till the % set in settings menu, now its ignoring this setting...

**Comment by Dzi4d3k** [comment]
> 
> 
> > Should be fixed again. That was my fault. Forgot about some sideeffect there
> 
> Thanks!
> Its almost fixed as it was... The trailers are filling up again and waiting till full... als waiting on the field...
> But the problem is that they wait till they are 100% full... before it was till the % set in settings menu, now its ignoring this setting...

This pretend to new issue.
